cmd/snap/cmd_find.go

```diff
@@ -100,6 +102,10 @@ func (x *cmdFind) Execute(args []string) error {
 		return ErrExtraArgs
 	}
 
+	if x.Positional.Query == "" {
+		return errors.New(i18n.G("you need to specify a query. Try \"snap find hello-world\"."))
```

Comment:
Though if we're using `snap` elsewhere, like you mentioned, it may not be worth the ugliness.